rmw_connextdds_common/src/common/rmw_impl.cpp

```diff
@@ -2973,6 +2994,14 @@ RMW_Connext_Service::send_response(
   rr_msg.gid.implementation_identifier = RMW_CONNEXTDDS_ID;
   rr_msg.payload = const_cast<void *>(ros_response);
 
+  DDS_Time_t timestamp;
```

Comment:
Since we are using Connext's C API we should initialize the variable to a known state, e.g.

```suggestion
  DDS_Time_t timestamp = DDS_TIME_INVALID;
```